### PR TITLE
CSI Driver configurations playbook

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
- CSI Driver configurations

Signed-off-by: Prajyot-Parab <prajyot.parab2@ibm.com>

### CODE REVIEW

File: examples/csi_driver_config_vars.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+region: ""              #IBM Cloud PowerVS service instance region
+zone: ""                #IBM Cloud PowerVS service instance zone
+
+masters:
+  - name: ""            #OCP Nodes Master name
+    id: ""              #OCP Nodes Master instance id
+
+workers:
+  - name: ""            #OCP Nodes Worker name
+    id: ""              #OCP Nodes Worker instance id
```
Comment (yussufsh) [inline examples/csi_driver_config_vars.yaml]: Can we have some example values in it? May be with multiple master nodes as well.

File: playbooks/csi_driver_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+- name: CSI Driver configuration
+  hosts: bastion
```
Comment (yussufsh) [inline playbooks/csi_driver_config.yaml]: Did you consider HA scenarios here with 2 bastion nodes?

File: playbooks/roles/csi-driver-config/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+# tasks file for csi driver configuration
+
+- name: Configure OCP Nodes
```
Comment (yussufsh) [inline playbooks/roles/csi-driver-config/tasks/main.yaml]: A better description?

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: examples/csi_driver_config_vars.yaml (hunk 1 above)
Comment (Prajyot-Parab) [reply examples/csi_driver_config_vars.yaml]: Done.

File: playbooks/csi_driver_config.yaml (hunk 2 above)
Comment (Prajyot-Parab) [reply playbooks/csi_driver_config.yaml]: Fixed, running on one bastion should be fine.

**Review by yussufsh — APPROVED** [review]
/lgtm
/cc @cs-zhang

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Pull request merged**